### DISCUSSION TITLE
URL Filter

### DISCUSSION THREAD

**Discussion by mvalezy** [will-stone/browserosaurus#650] [Ideas]
Hi, I love the app, and it's simplicity to use.

One cool feature for power users I would love to see is defining a default Browser for certain URL/domains.

I use Browserosaurus to separate my work stuff with my personal browsing.

Many thanks :)

**Comment by will-stone** [comment]
Thanks but this has already been covered: https://github.com/will-stone/browserosaurus/discussions/476

**Reply by mvalezy** [reply]
Thanks Will :)
